examples/color/color_cycle_default.py

```diff
@@ -13,7 +13,7 @@
 colors = prop_cycle.by_key()['color']
 
 lwbase = plt.rcParams['lines.linewidth']
-thin = float('%.1f' % (lwbase / 2))
+thin = lwbase / 2
```

Comment:
By curiosity, was there a purpose to the former "rounding to 0.1" scheme?

Reply:
Probably to make the title label look better(?), but I think 0.75 looks just fine too.

Reply:
Fair enough, and I think the same